nvm.sh

```diff
@@ -1022,6 +1022,12 @@ nvm_install_node_source() {
   local ADDITIONAL_PARAMETERS
   ADDITIONAL_PARAMETERS="$2"
 
+  local NVM_ARCH
+  NVM_ARCH="$(nvm_get_arch)"
+  if [[ $NVM_ARCH = *"arm"* ]]; then
```

Comment:
I believe this double bracket syntax is not portable - `nvm` has to work in POSIX, which includes `dash`, `sh`, `ksh`, `zsh`, and `bash`. Same, I believe, with the `+=` below.